README.md

```diff
@@ -16,9 +16,10 @@ helm repo add itzg https://itzg.github.io/minecraft-server-charts/
 You can then run `helm search repo itzg` to see the charts.
 
 ## Charts
-tt
-* [minecraft](https://github.com/itzg/minecraft-server-charts/tree/master/charts/minecraft)
-* [minecraft-bedrock](https://github.com/itzg/minecraft-server-charts/tree/master/charts/minecraft-bedrock)
+
+* [minecraft](./charts/minecraft)
```

Comment:
For these, can we use absolute URLs instead of relative URLs?  The reason for this is when this readme is presented in something like artifacthub, the links will still work.

Reply:
Will do